README.md

```diff
@@ -25,6 +25,11 @@ Clone/Update the [dapp-styles](https://github.com/ethereum/dapp-styles) in the `
 
 Starting the wall dapp using meteor
 
+    $ apt-get install nodejs-legacy
+    $ apt-get install npm 
+    $ npm update
+    $ npm install -g meteor
+    $ meteor update
```

Comment:
This is not necessary or wanted, as we are updating the packages/meteor. the right used version are in `.meteor/versions`, so it will automatically update when you run it after pull